services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> None:
+    user = get_user_model().objects.get(username=username)
+
+    order = Order(user=user)
+    order.save()
+
+    if date:
+        order.created_at = date
+        order.save()
+
+    for ticket in tickets:
+        Ticket.objects.create(movie_session_id=ticket["movie_session"],
+                              order=order,
+                              row=ticket["row"],
+                              seat=ticket["seat"])
+
+
+def get_orders(username: str = None) -> QuerySet[Order]:
```

Comment:
```suggestion
def get_orders(username: Optional[str] = None) -> QuerySet[Order]:
```
Because None as default

Reply:
Fix in all places